.changeset/fair-eels-march.md

```diff
@@ -0,0 +1,5 @@
+---
+'@urql/exchange-persisted': patch
```

Comment:
```suggestion
'@urql/exchange-persisted': major
```

As discussed, we'll have to mark this as a major, since it's a breaking change

Reply:
Indeed. Fixed in https://github.com/urql-graphql/urql/pull/3192/commits/7ea17f77112b7e595fb4a107abe650c18c711554.